pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -54,9 +58,30 @@ func (f *Makefile) SetTemplateDefaults() error {
 		f.Image = "controller:latest"
 	}
 
+	if f.ControllerRuntimeReleaseBranchName == "" {
```

Comment:
@mateusoliveira43 @varshaprasad96 

This part we will need to came back to use latest
we cannot relay on the branch. 
We need to relay on the tag

````
Downloading sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.17
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.17: sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.17: invalid version: unknown revision release-0.17
````

Reply:
yeah, checked and the last release is 0.17, but there is no branch to it. Probably need some change in controller-runtime to allow use of non-latest with setup-envtest